server/backend/database/database.go

```diff
@@ -143,6 +143,12 @@ type Database interface {
 	// after handling PushPull.
 	UpdateClientInfoAfterPushPull(ctx context.Context, clientInfo *ClientInfo, docInfo *DocInfo) error
 
+	// DeleteUserInfoByName deletes a user by name.
```

Comment:
How about moving this function near User related functions like `CreateUserInfo()`?

Reply:
Well it seems like the lines are still in the same position. Not planning on changing the lines?

Reply:
oh my mistake. I only moved methods related to the memory DB. Here is the commit: https://github.com/yorkie-team/yorkie/pull/934/commits/9ab114271c7ba2262b579e91cd04e5bc315e5cb8

I will fix it right now.